docs/source/models/other_models/alzheimers_population/index.rst

```diff
@@ -30,4 +30,230 @@
+  :math:`t`
+* :math:`X^\text{real}_{g,t}` = the real population corresponding to our
+  simulated population :math:`X_{g,t}`
+* :math:`\hat X_{g,t}` = the imagined total model population in group
+  :math:`g`, including people with and without AD, of which
+  :math:`X_{g,t}` counts the subset with AD
+* :math:`\hat X^\text{real}_{g,t}` = the total real population in group
+  :math:`g` at time :math:`t` according to GBD
+
+
+We need to determine :math:`X_{g,t_0}` (the initial simulated
+population) for each demographic group :math:`g`. Let :math:`p_{g,t}` be
+the prevalence of Alzheimer's disease and other dementias in demographic
+group :math:`g` at time :math:`t,` for a given location. Two relations
+among the above quantities are:
+
+.. math::
+
+  \begin{align*}
+  X_{g,t} = S \cdot X^\text{real}_{g,t}
+  \quad\text{and}\quad
+  X^\text{real}_{g,t} = p_{g,t} \cdot \hat X^\text{real}_{g,t}.
+  % X_{g,t} &= p_g \cdot \hat X_{g,t} \\
+  % \hat X_{g,t} & = S \cdot \hat X^\text{real}_{g,t}
+  \end{align*}
+
+(For :math:`t\ne t_0`, the first relation assumes that our simulated
+population accurately tracks the real-world population over time.)
+Therefore, at time :math:`t_0`,
+
+.. math::
+  :label: initial_pop_eq
+
+  X_{g,t_0}
+  = S \cdot X^\text{real}_{g,t_0}
+  = S\cdot p_{g,t_0} \cdot \hat X^\text{real}_{g,t_0}
+  = X_{t_0} \cdot \frac{p_{g,t_0}}{p_\text{Alz}}
+    \cdot \frac{\hat X^\text{real}_{g,t_0}}{\hat X^\text{real}_{t_0}},
+
+where the final equality follows from plugging in formula
+:eq:`model_scale_eq` for the model scale :math:`S`. This equation tells
+us how many simulants to initialize into each demographic group based on
+known parameters.
+
+.. note::
+
+  Another way to write :eq:`initial_pop_eq` is
+
+  .. math::
+
+    X_{g,t_0} = X_{t_0}
+    \cdot \frac{\text{# of real people in subgroup $g$ with Alzheimer's}}
+      {\text{# of real people in whole population with Alzheimer's}}.
+
+  Thus, we could compute :math:`X_{g,t_0}` using prevalence counts from
+  GBD instead of prevalence rates.
+
+  To verify that :eq:`initial_pop_eq` gives us the correct total number of
+  initial simulants, note that
+
+  .. math::
+
+    \begin{align*}
+    \sum_g X_{g,t_0}
+    = \sum_g X_{t_0}
+      \cdot \frac{p_{g,t_0} \cdot \hat X^\text{real}_{g,t_0}}
+      {p_\text{Alz} \cdot \hat X^\text{real}_{t_0}}
+    &= X_{t_0} \cdot \sum_g
+      \frac{X^\text{real}_{g,t_0}}{X^\text{real}_{t_0}} \\
+    &= X_{t_0} \cdot
+      \frac{\sum_g X^\text{real}_{g,t_0}}{X^\text{real}_{t_0}}
+    = X_{t_0} \cdot
+      \frac{X^\text{real}_{t_0}}{X^\text{real}_{t_0}}
+    = X_{t_0}.
+    \end{align*}
+
+Adding New Simulants
+++++++++++++++++++++
+
+Let :math:`N_{g,t}` denote the number of new simulants in demographic
+group :math:`g` that we want to add to the simulation at time :math:`t`.
+We will assume that :math:`N_{g,t}` is a Poisson random variable with
+mean :math:`\lambda_{g,t} \cdot \Delta t \cdot 1_{\{\text{simulation
```

Comment:
```suggestion
mean :math:`\lambda_{g,t} \cdot \Delta t \cdot 1_{\{\text{simulation 
```
maybe the space will help with a weird render thing, this is what it looks like now:
<img width="525" height="74" alt="image" src="https://github.com/user-attachments/assets/eeebb9ab-e057-4660-a699-d9f76077f79e" />

Reply:
All whitespace is equivalent in this context, so adding a space here doesn't help. The only way I've found to fix this is to keep the whole equation on one line, or put the line break in a different place. The problem is that I have a command to auto-wrap lines, so I have to remember not to auto-wrap paragraphs where I needed to use specific line breaks to render the math correctly. It looks like I forgot in this case -- I'll fix it.

Reply:
Fixed, for now.